.github/workflows/schedule.yaml

```
@@ -0,0 +1,38 @@
+---
+name: Schedule
+
+on:
+  schedule:
+    - cron: 15 11 * * *
```

Let's avoid peak hours and set to something like around 6 o'clock in the morning (CET)

```suggestion
    - cron: 4 4 * * *
```